conans/client/downloaders/download.py

```diff
@@ -1,9 +1,16 @@
+from conans.client.downloaders.remote_cache_downloader import RemoteCacheDownloader
 from conans.client.downloaders.cached_file_downloader import CachedFileDownloader
 from conans.client.downloaders.file_downloader import FileDownloader
 
 
-def run_downloader(requester, output, verify, config, user_download=False, use_cache=True, **kwargs):
+def run_downloader(requester, output, verify, config, user_download=False, use_cache=True,
+                   use_rt_cache=None, **kwargs):
     downloader = FileDownloader(requester=requester, output=output, verify=verify, config=config)
+
+    if use_rt_cache and config.sources_backup:
```

Comment:
This was confusing while I was reading it: you can call ``run_downloader(..., use_rt_cache=True,....)`` but the sources backup cache might not be used because ``config.backup_sources`` is not configured. Maybe this could be considered:
- Rename ``use_rt_cache`` => ``backup_sources_enabled`` or something like that. 
- Just use ``use_cache and config.sources_backup``

Just a very minor detail, if the UI could be improved, great, otherwise, not a big issue

Reply:
I was using the same approach as the download cache with `use_cache` argument and then `config.download_cache`. 

`use_cache` is also used in the context of the REST APIs, where `use_rt_cache` must be always False, so we need both of them.

Renaming, ok, but then let's rename both?
 * use_rt_cache => sources_backup_enabled
 * use_cache => download_cache_enabled